docs/pages/comment_parsing.rst

```diff
@@ -126,3 +131,75 @@ The full program
         if comment.body == "Hello" and comment.id not in already_done:
             comment.reply(' world!')
             already_done.add(comment.id)
+
+Navigating Trees
+------------------
+
+Comments are in trees, each submission has a comment forest. However with
+some simple manipulation, we can manipulate
+
+Special Comment Methods
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Going up and down comment trees is useful for various scripts and is sometimes
+needed, for replying to the parent or ancestral submitter, however natively
+there aren't any easy solutions. Due to this PRAW has implemented ways to
+do this from Comment objects.
+
+Comment objects have the following special methods:
+
+* ``get_tree()``
+* ``get_submission()``
+* ``get_context()``
+* ``get_parent()``
+
+PRAW can also return the various permalinks to the related "special" Submission
+objects.
+
+Please see :class:`.Comment` for more information as to how exactly to use
+these methods
+
+Parameter Passing and MoreComments Objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Any original parameters passed or functions done on the original Submission object
+are not remembered when getting these new "special" Submission objects. For that
+reason, you are allowed to both use the same ones when using the above methods or
+even change them, if that pleases you.
+
+Also, it is important to know that any original MoreComments objects that were
+replaced are "unreplaced" when using the above methods. It is probably better to
```

Comment:
I think this adds more confusion. It may be simple enough to say that the above methods make requests to reddit and return new _partial_ Submission objects. These objects may have comment trees containing `MoreComments` objects, even if they were already replaced in the original `Submission` object.